src/hooks/AuthHooks.res

```diff
@@ -110,6 +109,6 @@ let useApiFetcher = () => {
         )
       })
     },
-    [token],
+    [],
```

Comment:
`authStatus` should be in dependency array as it might change and you might need to get a new reference of function to be returned. 
Check once

Reply:
This callback is primarily used within the useGetMethod and useUpdateMethod hooks, rather than on its own and whenever it is called it will be initialised so it will have the updated values . Hence we dont need to add dependency

Reply:
How sure we are that authStatus value will always be there the correct one? I said because let's say someone use this function directly then it won't be having updated value

Reply:
That is the convention to not use this directly rather than use `useGetMethod` and `useUpdateMethod` hooks

Reply:
Sure then.